packages/server/test/e2e/3_issue_3658_spec.coffee

```diff
@@ -0,0 +1,13 @@
+e2e = require("../support/helpers/e2e")
+
+describe "e2e issue 3658", ->
+  e2e.setup()
+
+  ## https://github.com/cypress-io/cypress/issues/3658
+
+  it "passes", ->
+    e2e.exec(@, {
+      spec: "issue_3658_spec.js"
+      snapshot: true
+      expectedExitCode: 0
+    })
```

Comment:
Does this have to be an e2e test? I don't see any output in the snapshot that's necessary to prove this test is valid. I think we could move this into an test in the driver

Reply:
hmm, perhaps

Reply:
That would be nice since it would run in <1s vs 30 seconds